qiita_db/parameters.py

```diff
@@ -66,6 +66,56 @@ def create(cls, param_set_name, **kwargs):
 
         return cls(id_)
 
+    @classmethod
+    def iter(cls):
+        """Iterates over all parameters
+
+        Returns
+        -------
+        generator
+            Yields a parameter instance
+        """
+        conn_handler = SQLConnectionHandler()
+        sql = "SELECT {0} FROM qiita.{1}".format(cls._column_id, cls._table)
+
+        for result in conn_handler.execute_fetchall(sql):
+            yield cls(result[0])
+
+    @property
+    def name(self):
+        """The name of the parameter set
+
+        Returns
+        -------
+        str
+            The name of the parameter set
+        """
+        conn_handler = SQLConnectionHandler()
+        return conn_handler.execute_fetchone(
+            "SELECT param_set_name FROM qiita.{0} WHERE {1} = %s".format(
+                self._table, self._column_id),
+            (self.id,))[0]
+
+    @property
+    def values(self):
+        """The values of the parameter set
+
+        Returns
+        -------
+        dict
+            Dictionary with the parameter values keyed by parameter name
+        """
+        conn_handler = SQLConnectionHandler()
+        result = dict(conn_handler.execute_fetchone(
+            "SELECT * FROM qiita.{0} WHERE {1} = %s".format(
```

Comment:
This method belongs to a base class, so I don't know which are those columns. However, I know that the two columns that are below exist, and they are not needed, so I just remove them.